narwhals/_dask/expr.py

```diff
@@ -174,7 +178,7 @@ def func(df: DaskLazyFrame) -> list[dask_expr.Series]:
             function_name=f"{self._function_name}->{expr_name}",
             root_names=root_names,
             output_names=output_names,
-            returns_scalar=self._returns_scalar or returns_scalar,
+            returns_scalar=returns_scalar,
```

Comment:
the `returns_scalar=self._returns_scalar or returns_scalar` was quite strange...I was probably thinking that once we have an expression that returns a scalar, then everything after that keeps returning a scalar. But that's not necessarily true if we do broadcasting on the left (e.g. `nw.col('a').sum() + nw.col('b')`)

I think it's better if the caller of `_from_call` is responsible for setting `returns_scalar` correctly